apps/daily-scrum/src/app/daily-scrum/orgs/[orgId]/settings/members-settings-card.tsx

```diff
@@ -1,21 +1,35 @@
-import { Button } from "ui/button";
+import { buttonVariants } from "ui/button";
 import { Card, CardContent, CardHeader, CardTitle } from "ui/shadcn-ui/card";
 import MemberList from "./member-list";
+import Link from "next/link";
+import { getParams } from "next-impl-getters/get-params";
+import { cn } from "ui";
 
 type Props = {};
 
 // TODO: pass in data
 // TODO: restrict updatd to org owner and admin
 const MembersSettingsCard = (props: Props) => {
+  const params = getParams() as { orgId: string };
   return (
     <Card>
       <CardHeader>
         <div className="flex justify-between items-center">
           <CardTitle className="text-base">Members</CardTitle>
 
-          <Button variant="default" className="ml-auto" size="sm">
-            Invite member
-          </Button>
+          <Link
+            className={cn(
+              buttonVariants({
+                variant: "default",
+                size: "sm",
+              }),
+              "ml-auto"
+            )}
+            href={`/daily-scrum/orgs/${params.orgId}/settings/add-member`}
+            scroll={false}
+          >
+            Add member
+          </Link>
```

Comment:
A button link has been added to the members settings section.
<img width="1464" alt="image" src="https://github.com/kiwiy-is/dailyscrum/assets/17868599/b717ad93-b746-41dd-b845-ad265c1fac2a">